rest/rest-resources/src/main/java/org/apache/brooklyn/rest/resources/EntityResource.java

```diff
@@ -358,4 +362,42 @@ public List<Object>  getSpecList(String applicationId, String entityId) {
         List<SpecSummary> specTag = BrooklynTags.findSpecHierarchyTag(entity.tags().getTags());
         return (List<Object>) resolving(specTag).preferJson(true).resolve();
     }
+
+    @Override
+    public List<WorkflowExecutionContext> getWorkflows(String applicationId, String entityId) {
```

Comment:
I think this new endpoint as the same as the other should check for entitlements
```
// getWorkflows and replayWorkflow
        if (Entitlements.isEntitled(mgmt().getEntitlementManager(), Entitlements.SEE_ENTITY, entity)) {
            return EntityTransformer.entitySummary(entity, ui.getBaseUriBuilder());
        }
// replayWorkflow
        if (Entitlements.isEntitled(mgmt().getEntitlementManager(), Entitlements.REPLAY_WORKFLOW, entity)) {
            return EntityTransformer.entitySummary(entity, ui.getBaseUriBuilder());
        }

```
replayWorkflow could be also authorised by `Entitlements.INVOKE_EFFECTOR` instead creating a new entitlement